tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/functions/CustomFunctionTest.java

```diff
@@ -43,4 +43,12 @@ public void testDuplicateParameters() {
         BAssertUtil.validateError(compile, 0, "redeclared symbol 'param'", 1, 35);
     }
 
+    @Test(description = "Test defining ballerina function with extra parameters with missing comma")
+    public void testExtraParameters() {
+        CompileResult compile = BCompileUtil.compile("test-src/functions/extra-parameters.bal");
```

Comment:
How about something like

```suggestion
        CompileResult compileResult = BCompileUtil.compile("test-src/functions/extra-parameters.bal");
```